stages/org.osbuild.mkdir

```diff
@@ -3,23 +3,26 @@ import os
 import sys
 
 import osbuild.api
-from osbuild.util.path import in_tree
+from osbuild.util import parsing
 
 
-def main(tree, options):
+def main(args):
+    options = args["options"]
+
     for item in options["paths"]:
         path = item["path"]
         mode = item.get("mode", 0o777)
         parents = item.get("parents", False)
         exist_ok = item.get("exist_ok", False)
 
-        if not path.startswith("/"):
-            print("WARNING: relative path used, this is discouraged!")
-
-        target = os.path.join(tree, path.lstrip("/"))
-        if not in_tree(target, tree):
-            raise ValueError(f"path {path} not in tree")
+        if "://" not in path:
```

Comment:
(nitpick) most python projects seem to have `if not "://" in path:`  (and indeed below this form is used). IIRC this was even a pylint errorat some point, so maybe worth flipping for consistency.

Reply:
I use it because otherwise lint fails locally:
```
E713 [*] Test for membership should be `not in`
...
-        if not "://" in path:
+        if "://" not in path:
```

Reply:
Oh, interesting. Sorry for the noise then - then I misremembed /o\